exporter/datadogexporter/metrics_exporter_test.go

```diff
@@ -145,18 +144,18 @@ func Test_metricsExporter_PushMetricsData(t *testing.T) {
 						"tags":   []interface{}{"lower_bound:0", "upper_bound:inf", "env:dev"},
 					},
 					map[string]interface{}{
```

Comment:
I don't get why this test changes. I would expect the behavior of this test to be exactly the same as with the previous implementation, except for the faulty metric. Why do we change this?

Reply:
It’s only the order of the two metrics that is reversed. There is no change.